website/docs/components/app-footer/partials/code/component-api.md

```diff
@@ -0,0 +1,121 @@
+## Component API
+
+### AppFooter
+
+The base `AppFooter` component includes a copyright notice. It also wraps and contains the `AppFooter::LegalLinks`, `AppFooter::StatusLink`, `AppFooter::Link`, and `AppFooter::Item` child components as well as `ExtraBefore` and `ExtraAfter` yielded components.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="<[AF].extraBefore>" @type="yielded component">
+    Container that yields its content before the `<ul>` list of links and items.
+  </C.Property>
+  <C.Property @name="<[AF].Item>" @type="yielded component">
+    The generic `AppFooter::Item` component (see below).
+  </C.Property>
+  <C.Property @name="<[AF].Link>" @type="yielded component">
+    The `AppFooter::Link` component (see below).
+  </C.Property>
+  <C.Property @name="<[AF].extraAfter>" @type="yielded component">
+    Container that yields its content after the `<ul>` list of links and items.
+  </C.Property>
+  <C.Property @name="theme" @type="string"  @values={{array "light" "dark" }} @default="light">
+    Set the overall theme use by the component and its children.
+  </C.Property>
+  <C.Property @name="ariaLabel" @type="string" @default="&quot;Footer items&quot;">
+    Accepts a localized string. The `ariaLabel` value is applied to the list of `AppFooter` content.
+  </C.Property>
+  <C.Property @name="copyrightYear" @type="string" @default="currentYear">
+    Pass a custom year value for the `Copyright` instead of the default current year value.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### AppFooter::LegalLinks
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="hrefForSupport" @type="string" @default="&quot;https://www.hashicorp.com/support&quot;">
+    Override the default href value with a custom url value.
+  </C.Property>
+  <C.Property @name="hrefForTerms" @type="string" @default="&quot;https://www.hashicorp.com/terms-of-service&quot;">
+    Override the default href value with a custom url value.
```

Comment:
We said we would make it clear that is likely this link should be overwritten, because these are the ToS of the website